docs/content/docs/connectors/flink-sources/mysql-cdc.md

```diff
@@ -833,6 +833,30 @@ There are two places that need to be taken care of.
   * If no update operation is performed on the specified column, the exactly-once semantics is ensured.
   * If the update operation is performed on the specified column, only the at-least-once semantics is ensured. However, you can specify primary keys at downstream and perform the idempotence operation to ensure data correctness.
 
+#### Warning
```

Comment:
Could you please add a link of this warning to `scan.incremental.snapshot.chunk.key-column` in the `Connector Options` table?

`scan.incremental.snapshot.chunk.key-column` option need to be provided in this table.